### PR TITLE
add Chinese translation

### PR DESCRIPTION
**Pull request by squirrelsc**
init Chinese translation

### CODE REVIEW

File: zh_CN/docs/EnableAssessor.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# **使用评估器**
+
+评估器模块用于评估正在运行的尝试。 最常用的情况是提前中止尝试。如果尝试的中间结果不够好，则可提前终止。
```
Comment (xuehui1991) [inline zh_CN/docs/EnableAssessor.md]: “评估器模块用于评估正在运行的尝试” after reading this, I have no idea about "assessor"

File: zh_CN/docs/AdvancedNAS.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# 高级神经网络架构搜索教程
+
+目前，许多 NAS（Neural Architecture Search，神经网络架构搜索）算法都在尝试上使用了 **权重共享（weight sharing）** 的方法来加速训练过程。 例如，[ENAS](https://arxiv.org/abs/1802.03268) 与以前的 [NASNet](https://arxiv.org/abs/1707.07012) 算法相比，通过'*子模型间的参数共享（parameter sharing between child models）*'提高了 1000 倍的效率。 而例如 [DARTS](https://arxiv.org/abs/1806.09055), [Network Morphism](https://arxiv.org/abs/1806.10282), 和 [Evolution](https://arxiv.org/abs/1703.01041) 等算法也利用或者隐式的利用了权重共享。
+
+这是关于如何在 NNI 中启用权重共享的教程。
+
+## 尝试间的权重共享
```
Comment (xuehui1991) [inline zh_CN/docs/AdvancedNAS.md]: "尝试间的权重共享"???

File: zh_CN/docs/AdvancedNAS.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# 高级神经网络架构搜索教程
+
+目前，许多 NAS（Neural Architecture Search，神经网络架构搜索）算法都在尝试上使用了 **权重共享（weight sharing）** 的方法来加速训练过程。 例如，[ENAS](https://arxiv.org/abs/1802.03268) 与以前的 [NASNet](https://arxiv.org/abs/1707.07012) 算法相比，通过'*子模型间的参数共享（parameter sharing between child models）*'提高了 1000 倍的效率。 而例如 [DARTS](https://arxiv.org/abs/1806.09055), [Network Morphism](https://arxiv.org/abs/1806.10282), 和 [Evolution](https://arxiv.org/abs/1703.01041) 等算法也利用或者隐式的利用了权重共享。
+
+这是关于如何在 NNI 中启用权重共享的教程。
```
Comment (xuehui1991) [inline zh_CN/docs/AdvancedNAS.md]: "这是关于如何在 NNI 中启用权重共享的教程。"????

File: zh_CN/README.Makefile.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# Makefile 文件和安装配置
+
+NNI uses GNU make for building and installing.
+
+`Makefile` 提供标准的目标 `生成`、`安装` 和 `卸载`, 以及不同设置的安装对象：
+
+* `dev-easy-install`: 针对非专家用户，自动处理所有内容；
+* `dev-easy-install`: 针对专家用户，自动处理所有内容；
+* `install`: target for NNI normal users, which installs NNI by copying files;
```
[the remark this answers is not in the document]
Comment (squirrelsc) [reply zh_CN/README.Makefile.md]: It should be a new string, so it's not handled. will include it in next push.

File: zh_CN/docs/EnableAssessor.md (hunk 1 above)
Comment (squirrelsc) [reply zh_CN/docs/EnableAssessor.md]: how about like below, add assessor in words.

**使用评估器（Assessor）** 

评估器（Assessor）模块用于评估正在运行的尝试。 最常用的情况是提前中止尝试。如果尝试的中间结果不够好，则可提前终止。

File: zh_CN/docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+## 提交新功能建议
+
+- 在适配使用场景时，总会需要一些新的功能。 可以加入新功能的讨论，也可以直接提交新功能的拉取请求。
+
+- 在自己的 github 账户下 fork 存储库。 在 fork 后， 对于 add, commit, push, 或 squash (如需要) 等改动都需要详细的提交消息。 然后就可以提交拉去请求了。
+
+## 参与源代码和缺陷修复
+
+拉去请求需要标记正确的标签，表明是缺陷修复或是改进源代码。 所有代码都需要遵循正确的命名约定和代码风格。
```
Comment (xuehui1991) [inline zh_CN/docs/CONTRIBUTING.md]: "拉去请求需要标记正确的标签" ??

Comment (squirrelsc) [reply zh_CN/docs/CONTRIBUTING.md]: will fixed like:
拉取请求需要选好正确的标签，表明是缺陷修复还是功能改进。

File: zh_CN/README.Makefile.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (squirrelsc) [reply zh_CN/README.Makefile.md]: missed this file.

File: zh_CN/README.md [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+* [实现自定义评估器](examples/assessors/README.md)
+* [使用进化算法为阅读理解任务找到好模型](examples/trials/ga_squad/README.md)
+
+## **贡献**
+
+欢迎贡献代码或提交建议，可在 [GitHub issues](https://github.com/Microsoft/nni/issues) 跟踪需求和缺陷。
+
+推荐新贡献者从标有 **good first issue** 的简单需求开始。
+
+如要安装 NNI 开发环境，参考： [配置 NNI 开发环境](docs/SetupNNIDeveloperEnvironment.md)。
```
[the remark this answers is not in the document]
Comment (squirrelsc) [reply zh_CN/README.md]: replaced all nni to NNI

File: zh_CN/README.Makefile.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
+# Makefile 文件和安装配置
+
+NNI uses GNU make for building and installing.
+
+`Makefile` 提供标准的目标 `生成`、`安装` 和 `卸载`, 以及不同设置的安装对象：
+
+* `dev-easy-install`: 针对非专家用户，自动处理所有内容；
```
Comment (leelaylay) [inline zh_CN/README.Makefile.md]:
```diff
-* `dev-easy-install`: 针对非专家用户，自动处理所有内容；
+* `easy-install`: 针对非专家用户，自动处理所有内容；
```

File: zh_CN/README.md [hunk 8]
```diff
@@ -0,0 +1,217 @@
+<p align="center">
+<img src="https://microsoft.github.io/nni/docs/img/nni_logo.png" width="300"/>
+</p>
+
+* * *
+
+[![MIT 许可证](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://github.com/Microsoft/nni/blob/master/LICENSE) [![生成状态](https://msrasrg.visualstudio.com/NNIOpenSource/_apis/build/status/Microsoft.nni)](https://msrasrg.visualstudio.com/NNIOpenSource/_build/latest?definitionId=6) [![问题](https://img.shields.io/github/issues-raw/Microsoft/nni.svg)](https://github.com/Microsoft/nni/issues?q=is%3Aissue+is%3Aopen) [![缺陷](https://img.shields.io/github/issues/Microsoft/nni/bug.svg)](https://github.com/Microsoft/nni/issues?q=is%3Aissue+is%3Aopen+label%3Abug) [![拉取请求](https://img.shields.io/github/issues-pr-raw/Microsoft/nni.svg)](https://github.com/Microsoft/nni/pulls?q=is%3Apr+is%3Aopen) [![版本](https://img.shields.io/github/release/Microsoft/nni.svg)](https://github.com/Microsoft/nni/releases) [![进入 https://gitter.im/Microsoft/nni 聊天室提问](https://badges.gitter.im/Microsoft/nni.svg)](https://gitter.im/Microsoft/nni?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (leelaylay) [inline zh_CN/README.md]: I suggest not to translate it because either '缺陷' or '故障' seems weird.

Comment (squirrelsc) [reply zh_CN/README.md]: ok

File: zh_CN/docs/Builtin_Tuner.md [hunk 9]
```diff
@@ -0,0 +1,313 @@
...
+| **Grid Search** [(用法)](#GridSearch)                                                                                                         | 网格搜索会穷举定义在搜索空间文件中的所有超参组合。 网格搜索可以使用的类型有 choice, quniform, qloguniform。 quniform 和 qloguniform 中的数值 q 具有特别的含义（不同于搜索空间文档中的说明）。 它表示了在最高值与最低值之间采样的值的数量。  |
+| [Hyperband](https://github.com/Microsoft/nni/tree/master/src/sdk/pynni/nni/hyperband_advisor) [(用法)](#Hyperband)                            | Hyperband 试图用有限的资源来探索尽可能多的组合，并发现最好的结果。 它的基本思路是生成大量的配置，并运行少量的步骤来找到有可能好的配置，然后继续训练找到其中更好的配置。                                                           |
+| [Network Morphism](https://github.com/Microsoft/nni/blob/master/src/sdk/pynni/nni/networkmorphism_tuner/README.md) [(用法)](#NetworkMorphism) | Network Morphism 提供了深度学习模型的自动架构搜索功能。 每个子网络都继承于父网络的知识和形态，并变换网络的不同形态，包括深度，宽度，跨层连接（skip-connection）。 然后使用历史的架构和指标，来估计子网络的值。 最后会选择最有希望的模型进行训练。          |
+| **Metis Tuner** [(用法)](#MetisTuner)                                                                                                         | 大多数调参工具仅仅预测最优配置，而 Metis 的优势在于有两个输出：(a) 最优配置的当前预测结果， 以及 (b) 下一次 Trial 的建议。 它不进行随机取样。 大多数工具假设训练集没有噪声数据，但 Metis 会知道是否需要对某个超参重新采样。                      |
```
Comment (xuehui1991) [inline zh_CN/docs/Builtin_Tuner.md]: add one line here?

File: zh_CN/docs/FAQ.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+### 安装时出现 Segmentation Fault (core dumped)
+
+> make: *** [install-XXX] Segmentation fault (core dumped)
+
+可依次试试以下方法：
+
+* 更新或重新安装 Python 中的 pip： `python3 -m pip install -U pip`
+* 在安装 NNI 时，添加 `--no-cache-dir` 参数：`python3 -m pip install nni --no-cache-dir`
+
+### 作业管理错误：getIPV4Address() failed because os.networkInterfaces().eth0 is undefined.
```
[the remark this answers is not in the document]
Comment (squirrelsc) [reply zh_CN/docs/FAQ.md]: good catch, this file should be removed.

File: zh_CN/docs/GetStarted.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+  builtinTunerName: TPE
+  classArgs:
+    optimize_mode: maximize
+trial:
+  command: python mnist.py
+  codeDir: ~/nni/examples/trials/mnist-annotation
+  gpuNum: 0
+```
+
+因为此尝试代码使用了 NNI 标记的方法（参考[这里](../tools/annotation/README.md) ），所以 *useAnnotation* 为 true。 *command* 是运行尝试代码所需要的命令，*codeDir* 是尝试代码的相对位置。 命令会在此目录中执行。 同时，也需要提供每个尝试进程所需的 GPU 数量。
```
[the remark this answers is not in the document]
Comment (squirrelsc) [reply zh_CN/docs/GetStarted.md]: good catch, this file should be removed.

File: zh_CN/docs/FAQ.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (squirrelsc) [reply zh_CN/docs/FAQ.md]: fixed

**Pull request merged**